### PR TITLE
Vue menu permission

### PR DESCRIPTION
**Pull request by aliceDxr**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-cli/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/toolkits/pro/template/tinyvue/src/components/menu/index.vue [hunk 1]
```diff
@@ -38,183 +36,70 @@
...
   copyRouter.sort((a: RouteRecordNormalized, b: RouteRecordNormalized) => {
     return (a.meta.order || 0) - (b.meta.order || 0);
   });
-
   const userStore = useUserStore();
   const role = computed(() => userStore.role);
-  let treeData = ref(copyRouter);
-
+  const getMenuIcon = (id:string) => {
+    switch(id){
```
Comment (hello-yezi) [inline packages/toolkits/pro/template/tinyvue/src/components/menu/index.vue]: 格式化处理，整个文件统一处理下

File: packages/toolkits/pro/template/tinyvue/src/router/routes/modules/cloud.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ export default {
     locale: 'menu.cloud',
     requiresAuth: true,
     order: 8,
+    roles: ['admin', 'user'],
```
Comment (hello-yezi) [inline packages/toolkits/pro/template/tinyvue/src/router/routes/modules/cloud.ts]: 'admin'， 'user'建议定义成枚举类型

File: packages/toolkits/pro/template/tinyvue/src/components/menu/index.vue (hunk 1 above)
Comment (aliceDxr) [reply packages/toolkits/pro/template/tinyvue/src/components/menu/index.vue]: 已处理

File: packages/toolkits/pro/template/tinyvue/src/router/routes/modules/cloud.ts (hunk 2 above)
Comment (aliceDxr) [reply packages/toolkits/pro/template/tinyvue/src/router/routes/modules/cloud.ts]: 已处理

**Review by hello-yezi — APPROVED** [review]
LGTM

**Pull request merged**